### DISCUSSION TITLE
ask for license

### DISCUSSION THREAD

**Discussion by srxqds** [SixLabors/ImageSharp#2199] [Q&A] [answered]
Can we use it for free for internal tool development in commercial companies?

**Comment by antonfirsov** [comment, marked as answer]
2.* yes since it remains under Apache. For vnext (3.0+) the relevant clause of the [License ](https://github.com/SixLabors/ImageSharp/blob/main/LICENSE) is:

> Works in Source or Object form are licensed to You under the Apache License, Version 2.0 if.
> [...]
> You are consuming the Work as a Direct Package Dependency in the capacity of a For-profit company/individual with
     less than 1M USD annual gross revenue.
> [...]
> For all other scenarios, Works in Source or Object form are licensed to You under the Six Labors Commercial License
   which may be purchased by visiting https://sixlabors.com/pricing/.

There is no clause distinguishing internal tools from customer-visible products.
